test/test_performer.py

```diff
@@ -36,6 +36,7 @@ def test_performer_linear_attention_approximates_quadratic_attention(inputs, num
     approx_layer = Performer(attention_method='linear', supports=1000, **kwargs)
     exact_layer = Performer(attention_method='quadratic', **kwargs)
 
+    tf.random.set_seed(42)
```

Comment:
test fails roughly 1 out of 100 due to randomness, seeded to make sure this doesn't happen